### DISCUSSION TITLE
Delete account

### DISCUSSION THREAD

**Discussion by SvenKortekaas** [OpenSignLabs/OpenSign#87] [Q&A] [answered]
Once you created a account, via the "Thanks plan", how can a I delete that account and all my information from the server?

**Comment by nxglabs** [comment]
Hi @SvenKortekaas 
Currently we don't have an option to delete the account. You can drop an email from your registered account to support@opensignlabs.com to get it deleted by our support team.

Would appreciate if you could share the reason you are moving away from us so that we can improve our product :)

**Comment by andrew-opensignlabs** [comment, marked as answer]
Delete account functionality has since been enabled.
Watch this [video](https://www.youtube.com/watch?v=pY0T7U164Y4) for a step by step guide.
